gradle-plugin/src/main/groovy/mozilla/telemetry/glean-gradle-plugin/GleanGradlePlugin.groovy

```diff
@@ -251,6 +251,8 @@ except:
                 args "--allow-missing-files"
                 args "-f"
                 args "markdown"
+                args "--option"
+                args "introduction_extra=\"For another view of these metrics, see the [Glean Dictionary](https://dictionary.protosaur.dev/)\""
```

Comment:
It goes without saying that this is completely untested, but I don't see how this could be *too* wrong. 😄